packages/server/tests/AuthenticationSignUp.test.ts

```diff
@@ -62,10 +63,15 @@ test.group('Authentication sign up tests group', (group) => {
                 address: 'no-reply@adonisenprovence.com',
             });
 
-            assert.equal(
-                message.subject,
-                `Welcome ${userNickname}, please verify your email !`,
+            const emailVerificationObjectRegex = new RegExp(
+                `[.*].*Welcome.*${userNickname},.*please.*verify.*your.*email.*!`,
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**💭 thought:** ‏I think the square brackets at the beginning of the Regex do not behave the way you want. See https://regex101.com/r/cwVrtv/1.

The square brackets contain a list of characters that match. I think you should escape them, like that: https://regex101.com/r/CdgogL/1.

Reply:
I've been doing so and eslint was triggering `useless escaped chars` errors so I've been removing them
Should we:
- start the regex using .* ?
- escape them and update eslint ?

Reply:
Do you remember what was the Regex exactly? Because escaping characters here is absolutely not useless. 😮